.gitignore

```diff
@@ -1,5 +1,5 @@
 .dub
+dub.*
```

Comment:
@Lansatac, I added it to ignore `dub.selections.json`
And, actually, `.gitignore` will effect only new files in repository. 
So if there will be, for example, `dub.selections.json` as new one, it will be ignored,
but if some one provide changes to `dub.json`, which already exists, it will appear in unstaged files list.